### PR TITLE
Fix go get and update to v1.8.7

### PR DESCRIPTION
**Pull request by acheron2302**
- First i change the github.com/urfave/cli to gopkg.in/urfave/cli.v1, because if you download with go get outside $GOPATH, then it will use the latest version which is the version 2. But in v2 they change all of the type so it break when you try to install with go get outside $GOPATH

- Second we need to update to version 1.8.7 with git tag v1.8.7, so that when they use go get inside the $GOPATH it will clone from git tag and our v1.8.6 tag is commit 6 month ago without the new go 1.11 module. So to fix that problem we need to update the to new tag v1.8.7

Fix issue:
#1214 #1199

### CODE REVIEW

**Comment by Pharb** [comment]
Thanks a lot for the effort @acheron2302, it is greatly appreciated!

If I look at the documentation here, it mentions that the default should still be v1: https://github.com/urfave/cli#using-v1-releases

>  if you download with go get outside $GOPATH, then it will use the latest version which is the version 2

What is the reason to "go get" without the $GOPATH set?

According to the examples there, it should not be necessary to use gopkg.in anymore: https://blog.golang.org/v2-go-modules

Maybe I don't fully understand the issue, but what is the exact reason to switch the import to gopkg.in?

**Comment by acheron2302** [comment]
@Pharb In https://blog.golang.org/v2-go-modules, it said: 

> Before Go modules, gopkg.in allowed package maintainers to follow what we now refer to as the import compatibility rule

In ơgo module doc](https://github.com/golang/go/wiki/Modules#why-does-installing-a-tool-via-go-get-fail-with-error-cannot-find-main-module):
>Leave GO111MODULE unset (the default, or explicitly set GO111MODULE=auto), which results in friendlier behavior. This will give you Go 1.10 behavior when you are outside of a module and hence will avoid go get reporting cannot find main module.

So that mean if you are outside a module and then it will return to 1.10 behavior which mean downloading the latest version into src. So that mean it will download the v2 from urfave and everything go kaboom. So we need to change it to gopkg.in and it will fix the problem

The another solution i haven't mention is that: In the README.md you guys should change the download to: `GO111MODULE=on go get -u`. But the problem it's that with it it will download v1.8.6 from this github and 1.8.6 release is 7 month ago and it still use gopkg.lock which is very outdate. So you guys need to update the release. And the reason i change it to gopkg.in is because in case somebody run `go get -u ` outside then there won't be any error. But i think by update the release to v1.8.7 and change the way to download to `GO111MODULE=on go get -u` will work just fine without the need of gopkg.in

**Comment by muesli** [comment]
I think this may potentially break usage of Go modules, though, as the canonical path of urfave/cli changed back to github.com/urfave/cli. I think we still build within GOPATH (and without using Go modules) on TravisCI. Maybe it's a good time to change that now, so we also verify that the package versions specified in `go.mod` actually work with our builds?

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.8.6
+1.8.7
```
Comment (dominikschulz) [inline VERSION:1]: This is incremented on release only.
Our latest release is 1.8.6, so this should stay as is.

**Comment by dominikschulz** [comment]
Obsoleted by #1273.

**Pull request closed without merging**